modules/cake.js

```diff
@@ -0,0 +1,121 @@
+const Augur = require("augubor"),
+  moment = require("moment"),
+  u = require("../utils/utils");
+
+function celebrate() {
+  if (moment().hours() == 15) {
+    testBirthdays().catch(error => u.errorHandler(error, "Test Birthdays"));
+    testCakeDays().catch(error => u.errorHandler(error, "Test Cake Days"));
+  }
+}
+
+const tenureCache = new u.Collection();
+function tenure(n) {
+  if (tenureCache.has(n)) return tenureCache.get(n);
+  return tenure(n - 1);
+}
+
+async function testBirthdays() {
+  // Send Birthday Messages, if saved by member
+  try {
+    const guild = Module.client.guilds.cache.get(Module.config.snowflakes.ldsg);
```

Comment:
Snowflakes don't get appended onto `Module.config` anymore. To access them, include this at the top of the file:
```const sf = require("../config/snowflakes.json");```